src/abstractdataframe/reshape.jl

```diff
@@ -51,74 +51,67 @@ d1m = stack(d1, Not([:a, :b, :e]))
 d1s_name = stack(d1, Not([:a, :b, :e]), variable_name=:somemeasure)
 ```
 """
-function stack(df::AbstractDataFrame, measure_vars::AbstractVector{<:Integer},
-               id_vars::AbstractVector{<:Integer}; variable_name::Symbol=:variable,
+function stack(df::AbstractDataFrame,
+               measure_vars = findall(col -> eltype(col) <: Union{AbstractFloat, Missing},
```

Comment:
While we're at it, shouldn't this use `Real`?

Reply:
I have not designed it. I think the intention was to exclude integers (only take floats), as integers are often used as ids not as values.

Reply:
OK. Not sure I love that behavior but...

Reply:
Well - I would prefer not to have the default at all (we do not use such a pattern anywhere else in the package), but let us leave it as it does not hurt I think and removing it would be breaking.